code/game/objects/structures/crates_lockers/closets/walllocker.dm

```diff
@@ -222,5 +222,37 @@
 	dir = WEST
 
 /obj/structure/closet/walllocker_double/survival/east
+	pixel_x = 32
+	dir = EAST
+
+/obj/structure/closet/walllocker_double/gameroom
+	desc = "A wall mounted storage cabinet."
+	name = "Game Cabinet"
+	icon = 'icons/obj/closets/bases/wall_double.dmi'
+	closet_appearance = /decl/closet_appearance/wall_double
+	density = FALSE
+	anchored = TRUE
+	store_mobs = 0
+	wall_mounted = 1
+	plane = TURF_PLANE
+	layer = ABOVE_TURF_LAYER
+
+	starts_with = list(
+
+		)
+
+/obj/structure/closet/walllocker_double/north
```

Comment:
Forgot to subtype these so you're just overriding the ones further up.

Reply:
This isn't supposed to be here. Will remove later. Tired.

Reply:
Fixed